leros2_record_utils:
  ros__parameters:
    task_name: "test_task"
    button_index: 3
    bag_name_prefix: "lerobot_bag"
    topics:
      - "/joy"
      - "/lerobot_task"
      - "/lerobot_is_recording"
      - "/lerobot_event"
      - "/joint_states"
      - "/cartesian_motion_controller/current_pose"
      - "/cartesian_motion_controller/target_frame"
      - "/top_camera/rgb/color/rect/image/compressed"
      - "/left_camera/rgb/color/rect/image/compressed"
      - "/wrist_camera/wrist_camera/color/image_raw/compressed"
      
